hello-hwasan/app/src/main/cpp/native-lib.cpp

```diff
@@ -0,0 +1,64 @@
+#include <jni.h>
```

Comment:
Copyright header in source files (aiui they're optional in build files, but should be in source files)